packages/components/src/schema/types/table.ts

```diff
@@ -6,6 +6,9 @@ export type KoliBriSortDirection = 'ASC' | 'DESC' | 'NOS';
 export type KoliBriTableDataType = Record<string, unknown>;
 
 export type KoliBriTableCell = {
+	/**
+	 * @deprecated It should be working without this property, too. (needs a refactoring)
```

Comment:
Haben wir hierfür schon ein Follow-Up Ticket?
In #7015 sollten wir die Stelle vielleicht explizit ausnahmen, um den Umfang nicht zu sprengen.